packages/dataviz/src/components/LineChart/LineChart.component.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { LineChart as RLineChart, Line, CartesianGrid, ResponsiveContainer, Tooltip, XAxis, YAxis, Legend } from 'recharts';
+
+import { LineChartEntry, LineChartOptions, LineOptions } from './LineChart.types';
+
+import { CustomTooltip } from './LineChartTooltip.component';
+
+export interface LineChartProps {
+	data: LineChartEntry[];
+	lines: LineOptions[];
+	chartOptions: LineChartOptions;
+};
+
+function LineChart({ data, lines, chartOptions }: LineChartProps) {
+	const {
+		width,
+		height,
+		margin,
+		showGridLines,
+		xAxisOptions,
+		leftYAxisOptions,
+		rightYAxisOptions,
+		tooltip,
+		legend,
+	} = chartOptions;
+
+	return (
+		<ResponsiveContainer width={width || "100%"} height={height || "100%"} debounce={1}>
+		  <RLineChart
+			data={data}
+			margin={margin || {}}
+		  >
+			{ !!showGridLines && <CartesianGrid stroke="#F2F2F2" strokeDasharray="2" vertical={false} />}
```

Comment:
Could use one of those [gray values](https://talend.surge.sh/design-system/?path=/docs/design-tokens-design-tokens--page&globals=theme:light#grayscale) or use directly design token

Reply:
Those gray values are indeed deprecated. Sorry, we'll remove that part of the doc soon. The tokens are the source of truth: https://talend.surge.sh/design-tokens/?path=/docs/welcome--page